dimagi/utils/couch/undo.py

```diff
@@ -32,3 +35,13 @@ def soft_delete(self, domain_included=True):
             record.save()
             self.save()
             return record
+
+
+def is_deleted(doc):
+    """
+    Guess if a document is deleted. Returns False if we're not sure.
```

Comment:
well i think the fact that the document exists (which it has to if we're asking this function if it's deleted) implies that the default is not deleted.

the fact that this lives in a module that is designed around the principle of deleting things by changing the doc type in this way makes me think the behavior is ok. though if you have strong feelings about changing the name i'm happy to do that